test/token/CodexTitle.test.js

```diff
@@ -9,10 +11,28 @@ require('chai')
   .should()
 
 contract('CodexTitle', function (accounts) {
+  const metadata = {
+    hashedMetadata: {
+      name: web3.sha3('First token'),
+      description: web3.sha3('This is the first token'),
+      images: ['asdf'].map((image) => {
```

Comment:
Maybe just make this:

```solidity
images: [web3.sha3('image data')],
```

Since you're not using the un-hashed versions anywhere.

Reply:
👍